scamp/Makefile

```diff
@@ -91,7 +91,7 @@ $(BUILD_DIR)%.gas: %.s
 $(BUILD_DIR)boot_aplx.elf: $(BUILD_DIR)boot_aplx.gas $(BUILD_DIR)spinnaker.gas $(BUILD_DIR)sark.gas
 	$(MKDIR) $(BUILD_DIR)
 	$(AS) -I $(BUILD_DIR) -o $(BUILD_DIR)boot_aplx.o $<
-	$(GP)-ld -Tboot.lnk -static --no-gc-sections --use-blx -nostartfiles $(BUILD_DIR)boot_aplx.o -o $@
```

Comment:
Does it only work with this removed?  This has been there for some time... I can see it appears to work with it removed, though equally it could be that this file isn't used or something...

Reply:
It was failing when I left that in there. (See the build of the first commit on this branch.)

Reply:
The issue seems to be, as [described](https://sourceware.org/pipermail/binutils/2021-June/116826.html), that `ld` from the GNU toolchain has got more strict about what options it supports. Which is good I suppose. The option is used by `gcc` but not by `ld`.

Reply:
Ah I see! OK!